contracts/stellar-interchain-token-service/src/tests/deploy_interchain_token.rs

```diff
@@ -225,26 +225,16 @@ fn deploy_interchain_token_fails_with_zero_initial_supply_and_no_minter() {
     let minter: Option<Address> = None;
     let initial_supply = 0;
 
-    let token_id = client.mock_all_auths().deploy_interchain_token(
```

Comment:
the test is called `fails` but it succeeds, we need to review more carefully lol